aws/policy/compute.yaml

```diff
@@ -155,6 +155,7 @@ Statement:
       - autoscaling:TerminateInstanceInAutoScalingGroup
       - ec2:DeleteVolume
       - elasticloadbalancing:AddTags
+      - elasticloadbalancing:AddListenerCertificates
       - elasticloadbalancing:ApplySecurityGroupsToLoadBalancer
       - elasticloadbalancing:AttachLoadBalancerToSubnets
       - elasticloadbalancing:RemoveTags
```

Comment:
It has nothing to do with your change, but can you also please move this line `- elasticloadbalancing:RemoveTags` before `- elasticloadbalancing:RegisterTargets` to keep the alphabetical ordering?